src/streaming/controllers/PlaybackController.js

```diff
@@ -155,6 +155,10 @@ function PlaybackController() {
             } else {
                 eventBus.trigger(Events.PLAYBACK_SEEK_ASKED);
                 logger.info('Requesting seek to time: ' + time);
```

Comment:
Perhaps this logging here should take place after modifying time on the next rows?